client/mysqldump.c

```diff
@@ -145,7 +145,9 @@ static TYPELIB opt_system_types=
 static ulonglong opt_system= 0ULL;
 static my_bool insert_pat_inited= 0, debug_info_flag= 0, debug_check_flag= 0,
                select_field_names_inited= 0;
-static ulong opt_max_allowed_packet, opt_net_buffer_length;
+static ulong opt_max_allowed_packet;
+static double opt_max_statement_time= 0.0;
+static ulong opt_net_buffer_length;
```

Comment:
Thank you for submitting PR. 
Could you combine opt_net_buffer_length and opt_max_allowed_packet in the same line like it was before and have only opt_max_statement_time in a separate line ? Everything else looks ok.

Reply:
corrected